android/src/main/java/com/brentvatne/common/toolbox/ReactBridgeUtils.kt

```diff
@@ -0,0 +1,142 @@
+
+/*
+* Toolbox to safe parsing of <Video props
+* These are just safe accessors to ReadableMap
+*/
+
+object ReactBridgeUtils {
+    @JvmStatic
+    fun safeGetString(map: ReadableMap?, key: String?, fallback: String?): String? {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getString(key) else fallback
+    }
+
+    @JvmStatic
+    fun safeGetString(map: ReadableMap?, key: String?): String? {
+        return safeGetString(map, key, null)
+    }
+
+    @JvmStatic
+    fun safeGetDynamic(map: ReadableMap?, key: String?, fallback: Dynamic?): Dynamic? {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getDynamic(key) else fallback
+    }
+
+    @JvmStatic
+    fun safeGetDynamic(map: ReadableMap?, key: String?): Dynamic? {
+        return safeGetDynamic(map, key, null)
+    }
+
+    @JvmStatic
+    fun safeGetBool(map: ReadableMap?, key: String?, fallback: Boolean): Boolean {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getBoolean(key) else fallback
+    }
+
+    @JvmStatic
+    fun safeGetMap(map: ReadableMap?, key: String?): ReadableMap? {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getMap(key) else null
+    }
+
+    @JvmStatic
+    fun safeGetArray(map: ReadableMap?, key: String?): ReadableArray? {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getArray(key) else null
+    }
+
+    @JvmStatic
+    fun safeGetInt(map: ReadableMap?, key: String?, fallback: Int): Int {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getInt(key) else fallback
+    }
+
+    @JvmStatic
+    fun safeGetInt(map: ReadableMap?, key: String?): Int {
+        return safeGetInt(map, key, 0);
+    }
+
+    @JvmStatic
+    fun safeGetDouble(map: ReadableMap?, key: String?, fallback: Double): Double {
+        return if (map != null && map.hasKey(key!!) && !map.isNull(key)) map.getDouble(key) else fallback
+    }
+    @JvmStatic
+    fun safeGetDouble(map: ReadableMap?, key: String?): Double {
+        return safeGetDouble(map, key, 0.0);
+    }
+    /**
+     * toStringMap converts a [ReadableMap] into a HashMap.
+     *
+     * @param readableMap The ReadableMap to be conveted.
+     * @return A HashMap containing the data that was in the ReadableMap.
+     * @see 'Adapted from https://github.com/artemyarulin/react-native-eval/blob/master/android/src/main/java/com/evaluator/react/ConversionUtil.java'
+     */
+    @JvmStatic
+    fun toStringMap(readableMap: ReadableMap?): Map<String, String?>? {
+        if (readableMap == null) return null
+        val iterator = readableMap.keySetIterator()
+        if (!iterator.hasNextKey()) return null
+        val result: MutableMap<String, String?> = HashMap()
+        while (iterator.hasNextKey()) {
+            val key = iterator.nextKey()
+            result[key] = readableMap.getString(key)
+        }
+        return result
```

Comment:
Is this same as this ?
https://github.com/react-native-video/react-native-video/blob/abdf3988a9173924ba8007003c007b633a2c44f6/android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java#L449C5-L464C6

If yes, do we need the old one ?

Reply:
Thank you, I will remove the old one !

Reply:
Done, Thank you !